etc/examples/tutorial/cycling/five/suite.rc

```diff
@@ -1,19 +0,0 @@
-[meta]
```

Comment:
Oh noes! My suite `five`! Just kidding, will copy it somewhere else. By the way, the tutorial and examples will be re-written for Cylc 8?

Reply:
Sorry!  Yeah, I think we need to find another way of making examples, documentation, etc. available.  I suspect the old examples were never really used anyway, outside of the tutorial (which used some of them), and by you (suite five!).